static/src/javascripts/projects/common/modules/commercial/top-banner-below-container.js

```diff
@@ -13,24 +13,35 @@ define([
 ) {
 
     function init() {
+        // If you are not at front you can enjoy refreshing adfree experience
         if (typeof config.tests.topBannerPosition === 'undefined'
-            || !config.page.isFront
-            || detect.getBreakpoint() === 'mobile') {
+            || !config.page.isFront) {
 
             return false;
         }
 
         var containerIndex = 0,
             $adSlotWrapper = $.create('<div class="fc-container top-banner-below-container top-banner-ad-container top-banner-ad-container--desktop top-banner-ad-container--above-nav"></div>'),
-            $adSlot        = bonzo(createAdSlot('top-above-nav', 'top-above-nav')),
-            $containers    = $('.fc-container');
+            $containers    = $('.fc-container'),
+            $adSlot        = null;
 
-        if ($containers.length >= 2) {
-            return $adSlotWrapper
-                .append($adSlot)
-                .insertAfter($containers[containerIndex]);
-        }
+        // For mobile we want to add inline1 MPU after the first container
+        if (detect.getBreakpoint() === 'mobile' && config.tests.topBannerPosition) {
+            $adSlot = bonzo(createAdSlot('inline1', 'container-inline'));
+
+            if ($containers.length > 0) {
+                return $adSlot.insertAfter($containers[containerIndex]);
+            }
+        // For desktop we want to place topAboveNav after first container
+        } else {
```

Comment:
This else statement will be fulfilled if `breakpoint !== 'desktop'` **or** `breakpoint === 'mobile' && !config.tests.topBannerPosition`. Are you sure the latter is correct? I wonder if you can't remove the test on `topBannerPosition` since it's already been tested at the beginning of the function.